src/components/clusterConfiguration/NetworkConfigurationForm.tsx

```diff
@@ -164,10 +164,10 @@ const NetworkConfigurationForm: React.FC<{
             <StackItem>
               <ClusterWizardToolbar
                 cluster={cluster}
-                errors={errors}
+                formErrors={errors}
                 dirty={dirty}
                 isSubmitting={isSubmitting}
-                isNextDisabled={!canNextNetwork({ isValid, isSubmitting, cluster })}
+                isNextDisabled={!(isValid && dirty) && !canNextNetwork({ cluster })}
```

Comment:
If the form is dirty and valid, it should be possible to submit it no matter of former validation state.